types/sortablejs/index.d.ts

```diff
@@ -28,6 +29,15 @@ declare class Sortable {
      */
     static create(element: HTMLElement, options: Sortable.Options): Sortable;
 
+    /**
+     * Mounts a plugin to the `Sortable`
+     * @param plugins One or more of the sortable plugins.
+     */
+    // todo:
+    // + tests
+    // + declarations for the plugins. Plugins should extend a `Sortable.Plugin` class.
+    static mount(...plugins: any[]): void;
```

Comment:
It seems like a missed opportunity to not also add types now for the plugins. This is better than nothing, but still.

Reply:
It does indeed. I would love to but I'm not too familiar with all the plugins.

I plan to add more to this at a later date in another pull.